### ISSUE TITLE
Drop support for DataONE publishing

### ISSUE THREAD

**Issue by Xarthisius** [whole-tale/whole-tale#149]
Currently we are unable to update a significant portion of the stack due to: https://github.com/DataONEorg/d1_python/issues/40

Usage tracking shows limited activity (only people I can identify as devs, which could be probably tracked to testing) in both registering and publishing to test DataONE instances. The "Analyze in Whole Tale" button developed during the course of original WT NSF proposal was never adopted and deployed upstream.

> Note
Data currently registered, and therefore Tales using it, will continue to work.

**Comment by Xarthisius** [comment]
Also noting that publishing was only supported for dev instances of Metacat. We were not allowed to publish to any production environment.

**Comment by mbjones** [comment]
ok, thanks. Yeah, that python issue is a pain. And something we do need to get resolved on multiple fronts.  And I was frustrated that the Analyze in Whole Tale button never became reliable enough to deploy to production -- it was so close. Thanks for the clarification as to why it needs to be removed.

We still do have people with interest in Whole Tale from DataONE, most recently a new Army Corps project with interest in running containerized models against repository data. I talked with Bertram about this last year, although it didn't get the funding it needed. I hope that, were we able to get a project like this funded, we could revisit this (and invest in getting it working again).

**Comment by Xarthisius** [comment]
> ok, thanks. Yeah, that python issue is a pain. And something we do need to get resolved on multiple fronts. And I was frustrated that the Analyze in Whole Tale button never became reliable enough to deploy to production -- it was so close. Thanks for the clarification as to why it needs to be removed.

I'm gonna note that we're dropping only the publishing part. Analyze in WT (and registering from DataONE in general) is still working and will still work, cause @ThomasThelen and @amoeba implemented that using raw http requests (kudos to them for that decision).

**Comment by mbjones** [comment]
Thanks, that is valuable to know. Our DRP group might indeed take advantage of that feature.

**Comment by craig-willis** [comment]
> And I was frustrated that the Analyze in Whole Tale button never became reliable enough to deploy to production -- it was so close.

The fact that the button was never deployed on DataONE is a frustration for me as well, considering the effort put into the integration. That and we were only ever allowed to enable publishing to https://dev.nceas.ucsb.edu/. 

Was there ever any issue reported about the reliability issues? I assume this is from long ago?

**Comment by mbjones** [comment]
Yes, from when our team at UCSB was working on it. The problem from my perspective was that every deployed beta that I tried had an error that did not work properly in some fundamental way -- i think mostly in terms of publishing to DataONE.  My perspective was that Tommy got it 95% of the way there, but it wasn't ready for users as they would have had a bad experience. All of that was on me for not managing our part of the project well enough to deliver it, and I would still see my team finishing any work on this before we deployed it. Tommy has left our group for another position, so if we did some work on this it would be with a different developer. So, please don't take my comments badly -- I was just lamenting that I didn't quite get it there, and still have HOPE, so was disappointed to see code being removed that we might build on later. But I fully understand that you need to make the current app work with the current constraints, so don't let me get in the way of that viability.
